scripts/lib/erm/VR.lua

```diff
@@ -74,5 +74,18 @@ function VR:M6()
 	return lastPos
 end
 
+function VR:T(maxVal)
+	local rng = Rng.getDefault()
+	local r = rng:nextInt(maxVal)
+	
+	return self.v + r
+end
+
+function VR:R(maxVal)
+	local rng = Rng.getDefault()
```

Comment:
We also need a way to get rng with some fixed initial seed or rng from gamestate for VR:R. @alexvins, any ideas?

Reply:
We should extend ServerCbProxy API with `ServerCallback::getRNG`.

Reply:
Thank you. Will try to do it.

Reply:
Fixed this.